contracts/reflect/src/testing.rs

```diff
@@ -24,8 +24,8 @@ impl Querier for CustomQuerier {
             Ok(v) => v,
             Err(e) => {
                 return Err(SystemError::InvalidRequest {
-                    error: format!("Parsing QueryRequest: {}", e),
-                    request: Binary(bin_request.to_vec()),
+                    error: format!("Parsing query request: {}", e),
+                    request: bin_request.into(),
```

Comment:
In the error message, isn't it better to name the type we're parsing explicitly? I'd personally even go with "Parsing QueryRequest<CustomQuery>"

Reply:
edited above message because i didn't notice it treats `<text>` as an html tag

Reply:
I moved away from `QueryRequest` because the type is different in different places of the code because of the generic argument. I think it is nice to use the same text everywhere. The serde type we tried to parse is available in `e`:

> GenericErr { msg: "Querier system error: Cannot parse request: Parsing query request: Error parsing into type cosmwasm_std::query::QueryRequest<reflect::msg::CustomQuery>: Invalid type in: {\"custom\":{\"capital\":{\"text\":\"food\"}}}", backtrace: None }

Reply:
Ah, cool